worlds/subnautica/Options.py

```diff
@@ -33,12 +33,27 @@ def get_event_name(self) -> str:
 
 
 class CreatureScans(Range):
-    """Place items on specific creature scans.
+    """Place items on specific, randomly chosen creature scans.
     Warning: Includes aggressive Leviathans."""
     display_name = "Creature Scans"
     range_end = len(all_creatures)
 
 
+class AggressiveScanLogic(Toggle):
```

Comment:
```suggestion
class AggressiveCreatureScanLogic(Toggle):
```

This could be misunderstood, I think maybe this change might make it more clear. (Don't take the suggestion directly, remember to refactor/rename in the IDE.)